src/useViewModel.ts

```diff
@@ -9,7 +9,7 @@ type AnyModel = {};
 
 function useViewModelInternal<P extends ViewModelProps>(
   viewModel: IViewModel<P>,
-  props?: P
+  props?: P,
```

Comment:
Looks like pre-commit hook with `prettier` didn't work. I fixed it in `main` branch. Pls, rebase it and recommit your changes with `git reset --soft HEAD~2`, to let prettier do formatting.